index.js

```diff
@@ -195,7 +196,7 @@ module.exports = function typeDetect(obj) {
     }
   }
 
-  if (getPrototypeOfExists && (symbolExists === false || typeof obj[Symbol.toStringTag] === 'undefined')) {
+  if (getPrototypeOfExists && (!symbolToStringTagExists || typeof obj[Symbol.toStringTag] === 'undefined')) {
```

Comment:
I know this is a little nitpicky, but this was originally `=== false` because negating a boolean next to parens can be a little too easy to miss. So personally, I'd like this to be `symbolToStringTagExists === false`. However if you disagree let me know.